docker-compose.yml

```diff
@@ -0,0 +1,70 @@
+version: "3"
+
+services:
+  emintdnode0:
+    container_name: emintdnode0
+    image: "ethermint/ethermintnode"
```

Comment:
Is this image exists of you want to create the image from the `Dockerfile` itself?

Reply:
I want to create it from the Dockerfile